src/Symfony/Component/Security/composer.json

```diff
@@ -37,17 +37,12 @@
         "symfony/finder": "~2.3|~3.0.0",
         "symfony/polyfill-intl-icu": "~1.0",
         "symfony/routing": "~2.2|~3.0.0",
-        "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/validator": "~2.5,>=2.5.5|~3.0.0",
-        "doctrine/common": "~2.2",
-        "doctrine/dbal": "~2.2",
-        "psr/log": "~1.0",
         "symfony/expression-language": "~2.6|~3.0.0",
-        "symfony/ldap": "~2.8|~3.0.0"
+        "symfony/ldap": "~2.8|~3.0.0",
+        "psr/log": "~1.0"
```

Comment:
there is already a conflict in both cases (2.7 -> 2.8 and 2.8 -> master) that we have to resolve everytime so this change won't enhance not degrade anything here...